pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = 'canonicalwebteam.store-api'
-version = '1.0.5'
+version = '1.0.6'
```

Comment:
This is about the most clear breaking of the interface of this module that you could imagine. I'm pretty sure this needs to be version `2.0.0`.

Reply:
You are right!